internal/cypress/sauce/runner.go

```diff
@@ -78,31 +86,48 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+
+	log.Info().Msgf("Suites completed: %d in progress: %d", completed, inprogress)
 	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+		res := <-results
+		completed++
+		inprogress--
+		logSuite(completed, inprogress, res.suiteName, res.browser, res.job.Passed)
+
+		if res.err != nil {
+			assetContent, err := r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, resto.ConsoleLogAsset)
+			if err != nil {
+				log.Err(res.err).Str("suite", res.suiteName).Msg("Failed to get job asset.")
```

Comment:
@valentyn88 something to talk her wold be to decide the log level, if we fail to download an assets maybe its just a warning to the user in regards of that we wont show the cli results properly, it's an error on our side, not in the user tests. We could discuss this on Monday with the team.

Reply:
I changed to warning instead of error log level.